wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -351,6 +351,10 @@ Section headings must use levels 2 to 5. The section heading proceeding the [tit
 
 Section headings must not skip a heading level (i.e. do not go from a level 2 heading to a level 4 heading) and must not contain formatting or links.
 
+Every section has a unique *identifier* that can be used to create a direct link to that part of the page. The default identifier for a section is a [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) transformation of its text, but it can also be [set manually](#custom-identifiers).
+
+Translations must not modify section identifiers. [Custom identifiers](#custom-identifiers) will most likely need to be used to achieve parity with English articles.
```

Comment:
> I don't follow what the "huge pain" would be unless I'm missing some other consequence of this. there's no more work done in this way than we already do, it's just in a different place

> whole url in english makes sense to me but I don't think TicClick said anything about that

ah well in case it wasn't clear, this paragraph here mandates that all identifiers (i.e. url fragments) in translations must use their corresponding identifier from the english heading, so yes this would cause quite a bit of change all over the wiki

am i understanding this correctly? i'm not against it, but i wanted to point out that with this we're going to be spamming custom identifiers into every translation

Reply:
> i'm not against it, but i wanted to point out that with this we're going to be spamming custom identifiers into every translation

yeah I'd like to avoid that (also, *will most likely need to be used* isn't really a rule)

Reply:
yeah it suggests how to achieve what the rule mandates, although there's really only one way to do it

"Translations must not modify section identifiers" is the rule and covers what i said